test/spoke/unit/factories/TokenFactory.t.sol

```diff
@@ -6,69 +6,17 @@ import {IAuth} from "../../../../src/misc/interfaces/IAuth.sol";
 import {Root} from "../../../../src/common/Root.sol";
 
 import {ShareToken} from "../../../../src/spoke/ShareToken.sol";
-import {VaultKind} from "../../../../src/spoke/interfaces/IVault.sol";
 import {IShareToken} from "../../../../src/spoke/interfaces/IShareToken.sol";
 import {TokenFactory} from "../../../../src/spoke/factories/TokenFactory.sol";
 
 import "forge-std/Test.sol";
 
-import {BaseTest} from "../../BaseTest.sol";
-
 interface SpokeLike {
     function getShare(uint64 poolId, bytes16 scId) external view returns (address);
 }
 
 contract FactoryTest is Test {
-    uint256 mainnetFork;
-    uint256 polygonFork;
-    address root;
-
-    function setUp() public {
-        if (vm.envOr("FORK_TESTS", false)) {
-            mainnetFork = vm.createFork(vm.rpcUrl("ethereum-mainnet"));
-            polygonFork = vm.createFork(vm.rpcUrl("polygon-mainnet"));
-        }
-
-        root = address(new Root(48 hours, address(this)));
-    }
-
-    function testTokenFactoryIsDeterministicAcrossChains(bytes16 scId) public {
```

Comment:
IIUC, this test is no longer representative for us. Factories are deployed with `create3`, and the Token factory creates tokens with create2, fully independent of which chain it is, only parameters are `salt` and `decimals`